OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs

```diff
@@ -15,8 +15,8 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public abstract class SupportPowerInfo : PausableConditionalTraitInfo
 	{
-		[Desc("Measured in seconds.")]
-		public readonly int ChargeTime = 0;
+		[Desc("Measured in ticks.")]
+		public readonly int ChargeDelay = 0;
```

Comment:
Can we please call this `Interval` instead? Our naming convention is to use *Delay for one-off events, and Interval for things that repeat.